### ISSUE TITLE
changing the display of the number of players and the server status

### ISSUE THREAD

**Issue by scpsltest** [Exiled-Team/DiscordIntegration#147]
Change the unfortunate message "Playing, waiting for connection" with something more helpful such as "The server is offline" or "There are 20 players left". You can also include "Online players: 0/20" or "Players online: 0" in order to make it clearer. Additionally, it would be advisable to change this message in the configuration file (cfg) of your bot.
![image](https://github.com/Exiled-Team/DiscordIntegration/assets/152562652/89e400ea-1af9-43fc-a271-2e8b8e587b47)
![image](https://github.com/Exiled-Team/DiscordIntegration/assets/152562652/461f1478-63d8-4736-9223-25cd74c45a3c)

**Comment by lucasgames8957** [comment]
This is not possible with the current Discord API, please contact Discord about status messages for bots first.

**Comment by joker-119** [comment]
> This is not possible with the current Discord API, please contact Discord about status messages for bots first.

Wrong. He's simply asking for configurable messages for the status.

I'll consider adding this when i have time to do so, but it won't be a priority as it's not really going to be used by most people.

**Comment by lucasgames8957** [comment]
> > This is not possible with the current Discord API, please contact Discord about status messages for bots first.
> 
> Wrong. He's simply asking for configurable messages for the status.
> 
> I'll consider adding this when i have time to do so, but it won't be a priority as it's not really going to be used by most people.

Sorry about being incorrect, the reason I had that answer was that I have tried doing this once and it could only do "Watching", "Playing", or "Listening to" status messages. It could have been my terrible decision-making.
